### PR TITLE
AMP boilerplate changes to be done after validator changes to suppor…

### PR DESCRIPTION
**Pull request by sriramkrish85**
Amp Boiler plate changes to be done after validator changes to support the new boilerplate code is in.

### CODE REVIEW

**Comment by jmadler** [comment]
It might make sense to also create a corresponding issue in the docs repo to update the docs/samples over there

File: test/manual/intersection.amp.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
       padding-bottom: 75vh;
     }
   </style>
-  <style>body {opacity: 0}</style><noscript><style>body {opacity: 1}</style></noscript>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 1s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 1s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 1s steps(1,end) 0s 1 normal both;animation:-amp-start 1s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
```
Comment (cramforce) [inline test/manual/intersection.amp.html]: I just wanted to clarify my understanding: We decided on a 1s timeout?

Comment (sriramkrish85) [inline test/manual/intersection.amp.html]: Yes

Comment (dvoytenko) [inline test/manual/intersection.amp.html]: Hmm. Wasn't it 3 seconds?

**Comment by sriramkrish85** [comment]
PTAL

**Comment by sriramkrish85** [comment]
PTAL

**Comment by cramforce** [comment]
LGTM

**Comment by jridgewell** [comment]
Fixes https://github.com/ampproject/amphtml/issues/323.

**Pull request merged**